### PR TITLE
Updated some content and style

### PR DESCRIPTION
**Pull request by ShreyaTyagi0XD**
I tried to make it look better and I have made some changes in the content so that while reading it, both technical AND non-technical audience understands the concept in one go .

### CODE REVIEW

File: definitions/stateless_apps.md [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+- A Stateless application doesn’t save any client session (state) data on the server where the application lives.
+- Each session is carried out as if it was the first time.
+- Responses are not dependent upon data from a previous session.
+- Provides functionality to use print services, CDN (Content Delivery Network) or the Web Servers in order to process every short-term request.
+- For example, when you use a Coke machine, you’re the one who maintains the state. After buying a drink, the machine doesn’t remember what kind of drink you purchased, but you do because you’re holding it in your hand. In this respect, it’s more accurate to say that a stateless application externalizes its state rather than maintaining it on the server.
```
Comment (JasonMorgan) [inline definitions/stateless_apps.md:15]: No other entry is using bullet points. Please convert this section to a paragraph or two. Try and keep the whole section to no more than 6 sentences.

File: definitions/stateless_apps.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
 
 ### Problem it addresses
 
-Stateless applications tackle the problem of resiliency, because different pods across a [cluster](cluster.md) can work independently, with multiple requests coming to them at the same time. If there’s a problem, you can easily restart the application, and it will return to its initial state with little or no downtime. As such, the benefits of stateless applications include resiliency, elasticity, and high availability. However, most applications we use today are at least partly [stateful](stateful_apps.md), as they store things like preferences and settings to improve the user experience.
+Stateless applications tackle the problem of resiliency, elasticity, and high availability. Because different pods across a [cluster](cluster.md) can work independently, with multiple requests coming to them at the same time.
+If there’s a problem, you can easily restart the application, and it will return to its initial state with little or no downtime.
```
Comment (JasonMorgan) [inline definitions/stateless_apps.md:20]: feels like 19 and 20 can be in the same paragraph.

File: definitions/stateless_apps.md [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+Stateless applications tackle the problem of resiliency, elasticity, and high availability. Because different pods across a [cluster](cluster.md) can work independently, with multiple requests coming to them at the same time.
+If there’s a problem, you can easily restart the application, and it will return to its initial state with little or no downtime. 
+
+However, most applications we use today are at least partly [stateful](stateful_apps.md), as they store things like preferences and settings to improve the user experience. If your app requires more memory of what happens from one session to the next, however, stateful might be the way to go.
```
Comment (JasonMorgan) [inline definitions/stateless_apps.md:22]: I think we should remove this line.

File: definitions/stateless_apps.md [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
 
 ### How it helps
 
-Boiling everything down, in a Stateless Application the only thing your cluster is responsible for is the code, and other static content, being hosted on it. That’s it, no changing databases, no writes and no left over files when the pod is deleted. Stateless [containers](container.md) are easier to deploy, and you don’t need to worry about saving container data on persistent storage volumes. You also don't have to worry about backing up the data.
+In a Stateless Application the only thing your cluster is responsible for is the code, and other static content, being hosted on it. That’s it, no changing databases, no writes and no left over files when the pod is deleted. Stateless [containers](container.md) are easier to deploy, you don’t need to worry about saving container data on persistent storage volumes and you also don't have to worry about backing up the data.<br><br>
```
Comment (JasonMorgan) [inline definitions/stateless_apps.md:26]: drop the <br>'s and consider adding the last sentence back into the main paragraph. Alternatively you can just remove it.

**Review by JasonMorgan — CHANGES_REQUESTED** [review]
Great work! I like the changes in a general sense but there are some specific changes we'll need to merge it in.

File: definitions/stateless_apps.md (hunk 4 above)
Comment (ShreyaTyagi0XD) [reply definitions/stateless_apps.md]: I have made the changes, kindly review the file

**Comment by JasonMorgan** [comment]
Closing as this term has been OBE

**Pull request closed without merging**